GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.cs

```diff
@@ -118,12 +122,31 @@ public Task<IList<Bucket>> ListAllBucketsAsync(string project)
         /// <returns>A sequence of all buckets within the project.</returns>
         public IEnumerable<Bucket> ListBuckets(string project)
         {
+            return ListBuckets(project, null);
+        }
+
+        /// <summary>
+        /// Lists the buckets for a given project, synchronously but lazily.
+        /// </summary>
+        /// <remarks>
+        /// This method fetches the buckets lazily, making requests to the underlying service
+        /// for a page of results at a time, as required. To retrieve all the buckets in a single collection,
+        /// simply call LINQ's <c>ToList()</c> method on the returned sequence.
+        /// </remarks>
+        /// <param name="project">The project to list the buckets from. Must not be null.</param>
+        /// <param name="options">The options for the operation. May be null, in which case
+        /// defaults will be supplied.</param>
+        /// <returns>A sequence of all buckets within the project.</returns>
+        public IEnumerable<Bucket> ListBuckets(string project, ListBucketsOptions options)
```

Comment:
I'm curious now of the difference between List*All*BucketsAsync and ListBuckets, do we need the _All_ in the name?